### DISCUSSION TITLE
Problem when Syncing using 0.46.1 (build 16918)

### DISCUSSION THREAD

**Discussion by AKickIndieNutz** [GetPublii/Publii#1645] [Help]
Hi, I am not sure if anyone else has experienced the same error or has any suggestions as to a workaround, but help would be greatly appreciated...

I was using the previous Publii version 0.45.2 (build 16609) and using Simple Theme v 2.8.3.0. I decided to 'keep in step' with the cool new features and installed the latest version 0.46.1,as usual install worked fine. I then attempted to Sync my site, it seemed to run through as usual but halfway through pushing changes to Github I got an error Critical Error: Error Request Timed Out. I checked the logs and I am unable to see any problems except the error message. I re-ran and this time it seemed to suggest that the push to Github was successful, but when I check my Github Repo can see no new Commits,equally in Cloudflare no delpoyment has occured, so I am guessing it is not even getting as far as that. 

In frustration after many failed attempts I uninstalled and 're-wound' back to v 0.45.2, I repeated the Sync process and all seems to work fine it pushes up to Github and triggers to Cloudflare. Not sure if anyone has experienced or has any ideas how to resolve otherwise looks as though I will be stuck on an old version? 
Many thanks in advance

![2024-08-24_21h18_31](https://github.com/user-attachments/assets/1807a7ce-0157-4e13-b4cf-94f65d5af257)
![2024-08-24_21h19_32](https://github.com/user-attachments/assets/9866e32c-8b3e-4984-b96b-8bf32d1a5aeb)

**Comment by NatdanaiPuthom** [comment]
I have same issue as well, also when updating to 0.47 still same. Sometime it resolved after i restart and regenerate tokens but then it will stop working after a day or two. I never had this issue before in early 2025 and 2024.

**Comment by dozenhacker** [comment]
i have same issue.
first i get time out error.
try again. publii says your site is synced.
when i look in the repo, there are no files deployed.
i use publii 0.47.0 build 17290

**Reply by dozenhacker** [reply]
in the deployment-process.log it stops like this:
Git debug: git add done:
 Git debug: changes exists =  false

but it misses completion:
git commit done:
A local commit has been made (this is still only on your computer).

git push done:
The commit has been sent ("pushed") to your GitHub repository—only now will you see changes on GitHub.

**Reply by dozenhacker** [reply]
i tried with a new publii site made from scratch. no plugins. just one blogpage.
it was successful!
first i tried to sync a page i built with plugins using.  and it failed. maybe it is a timeout problem on the side of github?
